packages/slice-machine/lib/builders/CustomTypeBuilder/index.tsx

```diff
@@ -2,18 +2,22 @@ import { Box } from "theme-ui";
 
 import TabZone from "./TabZone";
 import { Tabs } from "./Layout";
+import { CustomTypeSM } from "@lib/models/common/CustomType";
 
-const CustomTypeBuilder = () => {
+type CustomTypeBuilderArgs = {
```

Comment:
```suggestion
type CustomTypeBuilderProps = {
```

It's props here right?